### DISCUSSION TITLE
Collections lose some reactivity after create

### DISCUSSION THREAD

**Discussion by TomSmedley** [payloadcms/payload#830] [Feature Requests & Ideas]
When you are first creating a title, you can see the large heading update as you type. After publishing it no longer retains that functionality. This could be fine as it could be seen as it's showing the current title until save. However, saving the collection item doesn't update it either which could cause confusion. The same applies to the `beforeValidate` hook, we are not seeing any update to the field unless we refresh the page.

In an example below, I have created a slug field that checks whether it has a value. If it doesn't it creates a slug from the title.
https://www.loom.com/share/044c1d1f7cb146c5a184ef1b2e8ce8bb

**Comment by jmikrut** [comment]
Hey @TomSmedley — we should be able to improve this quite easily. 

Let me tag this as `planned` and we'll make it happen.

Thanks for bringing this up!

**Reply by jmikrut** [reply]
Hey @TomSmedley — this is now fixed and merged into master!
